reconcile/utils/oc.py

```diff
@@ -373,6 +359,46 @@ def _msg_to_process_reconcile_time(self, namespace, resource):
             is_log_slow_oc_reconcile=self.is_log_slow_oc_reconcile,
         )
 
+    def process(self, template, parameters=None):
+        if parameters is None:
+            parameters = {}
+        parameters_to_process = [f"{k}={v}" for k, v in parameters.items()]
+        cmd = [
+            "process",
+            "--local",
+            "--ignore-unknown-parameters",
+            "-f",
+            "-",
+        ] + parameters_to_process
+        result = OCDeprecated._run(
```

Comment:
now that `process()` is back within `OCDeprecated`, I believe this can be changed back to original `self._run()`